server/athenian/api/align/queries/teams.py

```diff
@@ -68,21 +68,33 @@ def build_team_tree_from_rows(team_rows: Iterable[Mapping[str, Any]],
     return nodes, root_team_id
 
 
-async def _get_team_tree(
+async def fetch_team_tree(
     account: int, root_team_id: Optional[int], sdb: DatabaseLike,
 ) -> TeamTree:
     """Build the TeamTree for the Team root_team_id."""
     team_select = [Team.id, Team.parent_id, Team.name, Team.members]
     team_rows = await fetch_teams_recursively(
         account, sdb, team_select, [root_team_id] if root_team_id else None,
     )
-    nodes, root_team_id = build_team_tree_from_rows(team_rows, root_team_id)
-    return _build_team_tree(nodes[root_team_id], nodes)
+    return build_team_tree_from_rows(team_rows, root_team_id)
 
 
-def _build_team_tree(team_info: Dict[str, Any], all_teams: dict) -> TeamTree:
+def build_team_tree_from_rows(rows: Sequence[Row], root_team_id: Optional[int]) -> TeamTree:
```

Comment:
fixed nomenclature among fetch/build and tree/nodes, and added this new entrypoint